src/coreclr/gc/gcenv.ee.standalone.inl

```diff
@@ -316,7 +316,7 @@ inline void GCToEEInterface::DiagAddNewRegion(int generation, uint8_t* rangeStar
 
 inline void GCToEEInterface::LogErrorToHost(const char *message)
 {
-    if (g_runtimeSupportedVersion.MajorVersion >= GC_INTERFACE2_MAJOR_VERSION)
+    if (g_runtimeSupportedVersion.MajorVersion >= EE_INTERFACE_MAJOR_VERSION)
```

Comment:
this should be `>= 1`, shouldn't it? the `EE_INTERFACE_MAJOR_VERSION` from GC side should really be "the highest version this clrgc.dll calls" so if EE bumped up this version to 2 and added another method like LogErrorForGC (size_t gc_index) and this clrgc.dll is modified to call it, this code would become

```cplusplus

if (g_runtimeSupportedVersion.MajorVersion >= 1)
{
    g_theGCToCLR->LogErrorToHost(message);

    if (g_runtimeSupportedVersion.MajorVersion >= 2)
    {
         g_theGCToCLR->LogErrorForGC (gc_heap::settings.gc_index);
    }
}
```

Reply:
Ah, that's right.